proposal/09510/README.rst

```diff
@@ -48,17 +48,17 @@ Engineering infrastructure
 -Version Control : 
     Git
 -Test : 
-    C+++ : Google Tesr
+    C++ : Google Tesr
     
     Python : pytest
 
 Schedule 
 ====================================
-- Week 1: Implement algo. part on C++
-- Week 2: implement cuda part on C++
-- Week 3: implement cuda part on C++
-- Week 4: implement cuda part on C++
-- Week 5: Integrate the c++ and python code
+- Week 1: Implement decision tree class/algo. on C++
+- Week 2: Implement decision tree class/algo. on C++
+- Week 3: Implement decision forest class/algo. on C++
+- Week 4: Implement decision forest class/algo. on C++
+- Week 5: Compared the C++ version and python version
 - Week 6: Test the system
```

Comment:
Please write (automatic) tests as soon as possible.  It will help your productivity.